app/services/like_service.rb

```diff
@@ -25,6 +31,14 @@ def find_for_post(post_id)
     user ? likes.order(Arel.sql("author_id = #{user.person.id} DESC")) : likes
   end
 
+  def find_for_comment(comment_id)
+
+    comment = comment_service.find_by_id!(comment_id)
+    post = post_service.find!(comment.post.id) #checks implicit for public posts
```

Comment:
- Lint/UselessAssignment: Useless assignment to variable - `post`.
- Layout/LeadingCommentSpace: Missing space after `#`.